status.sh

```diff
@@ -185,43 +185,46 @@ Service_Server_Status_client() {
 }
 Installation_dependency() {
   mode=$1
-  [[ -z ${mode} ]] && mode="server"
-  if python --help >/dev/null 2>&1; then
+  if python3 --help >/dev/null 2>&1; then
+    ln -s /usr/bin/python3 /usr/bin/python 
+    python_status=1
+  elif python --help >/dev/null 2>&1; then
     python_status=1
-  elif python3 --help >/dev/null 2>&1; then
-    ln -s /usr/bin/python3 /usr/bin/python
-    python_status=2
   else
     python_status=0
   fi
   if [[ ${mode} == "server" ]]; then
     if [[ ${release} == "centos" ]]; then
       yum -y update
       if [ ${python_status} -eq 0 ]; then
-        yum -y install python unzip vim make
+        yum -y install python3 unzip vim make
+        ln -s /usr/bin/python3 /usr/bin/python
         yum -y groupinstall "Development Tools"
-      else
+      elif [ ${python_status} -eq 1 ]; then
         yum -y install unzip vim make
         yum -y groupinstall "Development Tools"
       fi
     else
       apt-get update -y
       if [ ${python_status} -eq 0 ]; then
-        apt-get -y install python unzip vim build-essential
-      else
+        apt-get -y install python3 unzip vim build-essential
+        ln -s /usr/bin/python3 /usr/bin/python
+      elif [ ${python_status} -eq 1 ]; then
         apt-get -y install unzip vim build-essential
       fi
     fi
-  else
+  elif [[ ${mode} == "client" ]]; then
     if [ ${release} == "centos" ]; then
       if [ "${python_status}" -eq 0 ]; then
         yum -y update
-        yum -y install python
+        yum -y install python3
+        ln -s /usr/bin/python3 /usr/bin/python
       fi
-    else
+    elif [ ${release} == "debian" ] || [${release} == "ubuntu"]; then
```

Comment:
It will boom since there is no space between the words and brackets

Reply:
BTW, I think it's better to use `else` instead of using `elif` there.

Reply:
After some thought on my part, I think you are right, as Debian and Ubuntu may not be the only ones using APT as a package manager, thank you very much for the correction.